src/main/scala/org/hombro/maze/MazeApp.scala

```diff
@@ -22,7 +22,7 @@ object MazeApp {
 
     dom.document.body.appendChild(canvas)
 
-    gameStart(canvas, context)
+    gameStart2(canvas, context)
```

Comment:
I'd rather we add some button switch on the page so we can compare maybe the speed of the impls since I'm gonna fix mine shortly. If you don't wanna add it, then I'll do it.